AISKU/scripts/uploadFilesToAzureStorage.ps1

```diff
@@ -0,0 +1,62 @@
+$storageAccountName = "daprodcdn";
+$containerName = "next";
+
+# get js sdk directory
+$jsSdkDir = Split-Path (Split-Path $MyInvocation.MyCommand.Path) -Parent;
```

Comment:
also, write wrappers for next and beta. Eventually, you want to be able to do three deployments one by one for flighting. beta, next, scripts/xxx (maybe scripts/b)